src/MensaWidget/MWMeal.class.st

```diff
@@ -171,6 +202,33 @@ MWMeal >> getSpacerMorph [
 		yourself
 ]
 
+{
+	#category : #star,
+	#'squeak_changestamp' : 'LC 6/23/2024 19:13'
+}
+MWMeal >> getStarMorph [
+	
+	^ PolygonMorph new 
+		color: self getStarMorphColor;
+		borderColor: Color fern lighter;
+		borderWidth: self class starBoarderWidth;
+		setVertices: self class getStarVertices;
+		on: #mouseDown send: #toggleLike: to: MWFavorites;
+		yourself
+]
+
+{
+	#category : #star,
+	#'squeak_changestamp' : 'LC 6/23/2024 19:05'
+}
+MWMeal >> getStarMorphColor [
+
+	(MWFavorites favouriteFoods includes: self name) 
+		ifTrue: [^ MWColorTheme secondaryColor] 
+		ifFalse:[^ Color transparent]
```

Comment:
missing space... I think it is cleaner to use iftrue and iffalse like this, but then the other methods have to be adapted

Reply:
yes I changed it but if feels so much cleaner

Reply:
i actually didn't mean to change it, we just have to agree on something

Reply:
yes but I unified it for now and we have to clafiy as a team on a standard then if we change it everything will be changed simulatniously and it is always uniform